packages/now-static-build/src/frameworks.ts

```diff
@@ -262,4 +262,23 @@ export default [
       },
     ],
   },
+  {
+    name: 'VuePress',
+    dependency: 'vuepress',
+    getOutputDirName: async () => 'docs/.vuepress/dist',
```

Comment:
```suggestion
    getOutputDirName: async () => '.vuepress/dist,
```

The documentation says the default output directory is `.vuepress/dist`.

https://v1.vuepress.vuejs.org/guide/getting-started.html#inside-an-existing-project